docs/api/python/relay/index.rst

```diff
@@ -27,16 +27,22 @@ compiler stack.
 .. toctree::
    :maxdepth: 2
 
+   analysis
    backend
-   base
-   build_module
-   expr
    frontend
    image
-   analysis
    transform
    nn
    op
-   scope_builder
    vision
    testing
+
+.. automodule:: tvm.relay
+    :members:
+    :imported-members:
+    :exclude-members: RelayExpr, Pass, PassInfo, function_pass, PassContext,
```

Comment:
We can also consider allow these members, but modify https://github.com/apache/incubator-tvm/blob/master/docs/conf.py#L254 to mark them as alias

Reply:
Yeah, I tried this before. But there are a lot of places that directly use the classes without namespace. It causes many ambiguous reference. For example:

https://github.com/apache/incubator-tvm/blob/master/python/tvm/ir/adt.py#L82